src/pages/Root/Root.js

```diff
@@ -52,7 +52,7 @@ const Root = ({
 		const { exchangeFeeRate, networkPrices, frozenSynths } = await getExchangeData(synths);
 		setExchangeFeeRate(exchangeFeeRate);
 		setNetworkGasInfo(networkPrices);
-		updateFrozenSynths(frozenSynths);
+		updateFrozenSynths({ frozenSynths });
```

Comment:
Do you want to move the fetch logic into redux soon or are you happy with that for now?